website/docs/r/project.html.markdown

```diff
@@ -26,8 +36,24 @@ resource "mongodbatlas_project" "my_project" {
 * `name` - (Required) The name of the project you want to create. (Cannot be changed via this Provider after creation.)
 * `org_id` - (Required) The ID of the organization you want to create the project within.
 
-~> **NOTE:** Project created by API Keys must belong to an existing organization.
+### Teams
+Teams attribute is optional
+
+~> **NOTE:** Atlas limits the number of users to a maximum of 100 teams per project and a maximum of 250 teams per organization.
+
+* `team_id` - (Required) The unique identifier of the team you want to associate with the project. The team and project must share the same parent organization.
 
+* `role_names` - (Required) Each string in the array represents a project role you want to assign to the team. Every user associated with the team inherits these roles. You must specify an array even if you are only associating a single role with the team.
+
+The following are the valid roles and their associated mappings:
```

Comment:
We don't have the mappings here like above, just the roles so I'd just change to:
The following are valid roles: